### ISSUE TITLE
Jest is unable to find module 'react-google-button'

### ISSUE THREAD

**Issue by kaladivo** [prescottprue/react-google-button#28]
**Do you want to request a *feature* or report a *bug*?**  
I want to report a bug

**What is the current behavior?**
When I run jest i get the following `Jest is unable to find module 'react-google-button'`

**What is the expected behavior?**
The package should not break jest tests

**Which versions of dependencies, and which browser are affected by this issue? Did this work in previous versions or setups?**
Jest 4.2.4. Create react app with typescript

**Steps to reproduce and if possible a minimal demo of the problem via [codepen](https://codepen.io/) or similar.**
You can reproduce the error by cloning this repo: https://github.com/kaladivo/jest-error
then run `yarn install && yarn test`. 

See this question on stackoverflow https://stackoverflow.com/questions/60024292/jest-is-unable-to-find-module-from-test-file/60025441#60025441. This seems to be caused by many reasons, one of which seems to be using newer import syntax. I am not sure how to fix this (otherwise I would made a PR), but maybe someone else can... It would be great if this module wont break the tests...

**Comment by prescottprue** [comment]
I have used this within other tests in the past, so I'll have to see what it might be - thanks for providing the repo where the error can be reproduced

**Comment by juliusdejon** [comment]
Same here when running my test
![image](https://user-images.githubusercontent.com/33895507/74510657-35a8a480-4f3f-11ea-8781-91cc255d0644.png)

**Comment by davidglivar** [comment]
I'm using create-react-app and ran into this as well. I'm working around the issue by mocking this module.

```tsx
// src/components/__mocks__/react-google-button.tsx
import React from 'react';
const GoogleButton = () => <div />;
export default GoogleButton;
```

more on mocking: https://jestjs.io/docs/en/manual-mocks

**Comment by vammu920** [comment]
@davidglivar  what to do after this step .

**Comment by prescottprue** [comment]
I find it interesting that it is looking for a jsx file in the posted error. It also says that support can be added for other extensions like `.js` to the `moduleFileExtensions`setting, which makes sense especially because this repo is using that extension instead of .jsx.

Not sure how much can be done on the library side other than switching to typescript or outputing a .tsx file. Has anyone experienced this when using any other setup different from CRA + typescript?

**Comment by juliusdejon** [comment]
I gave up and created my custom google-btn

**Comment by tervay** [comment]
I ran into this issue and I solved it by changing

```
import GoogleButton from "react-google-button";
```

to

```
import GoogleButton from "react-google-button/dist/react-google-button";
```

**Comment by umutlaguler** [comment]
@tervay it's not working on my project

**Comment by bdunks** [comment]
For others finding this issue, note that you can also solve this by explicitly importing a previous version (`"react-google-button": "^0.6.0"`) until the package.json main path is resolved.

**Comment by prescottprue** [comment]
Should be solved by updating to [v0.7.2](https://github.com/prescottprue/react-google-button/releases/tag/v0.7.2)
